lib/auth/webauthncli/api.go

```diff
@@ -184,6 +215,11 @@ func Register(
 		return FIDO2Register(ctx, origin, cc, prompt)
 	}
 
+	if winwebauthn.IsAvailable() {
```

Comment:
Move this before FIDO2? It seems like we'd like it to have preference regardless.